wormhole-connect/src/contexts/TokensContext.tsx

```diff
@@ -104,21 +105,34 @@ export const TokensProvider: React.FC<TokensProviderProps> = ({ children }) => {
     [],
   );
 
-  const tokenPricesToFetch: Set<TokenId> = new Set();
+  const tokenPricesToFetch: TokenMapping<boolean> = new TokenMapping();
```

Comment:
I think this should be a state variable like `tokenPrices`

Reply:
seems to work as-is but I can make it one